three.c

```diff
@@ -0,0 +1,90 @@
+#include <stdio.h>
+#include <math.h>      //This program determines whether three numbers could form a tringle and what kind of triangle they would form
+#include <stdlib.h>
+
+int isTriangle(int a, int b, int c)
+{
+	if (a+b<c)
+	{
+		printf("This is not a valid triangle, restart the program\n");
+	}
+	if (a+c<b)
+	{
+		printf("This is not a valid triangle, restart the program\n");
+	}
+	if (b+c<a)
+	{
+		printf("This is not a valid triangle, restart the program\n");
+	}
+	else
```

Comment:
Aside from this, you didn't handle input correctly and left three warnings. The reason for your three warnings was that your three functions didn't have matching return types. (they were written with int's but didn't return an int). Also, when an invalid triangle was put, it always returned as a scalene, which shouldn't have been done. You should have written your code such that if it's not a valid triangle, it would terminate the program after saying  so.